Composer/packages/client/src/recoilModel/dispatchers/publisher.ts

```diff
@@ -105,7 +105,6 @@ export const publisherDispatcher = () => {
     // remove job id in publish storage if published
     if (status === PUBLISH_SUCCESS || status === PUBLISH_FAILED) {
       const publishJobIds = publishStorage.get('jobIds') || {};
-      delete publishJobIds[`${projectId}-${target.name}`];
       publishStorage.set('jobIds', publishJobIds);
     }
```

Comment:
We should probably remove this whole block. What's the point of keeping it without the delete?

Reply:
Yes. It doesn't make sense.